Fleury/fleury.py

```diff
@@ -0,0 +1,32 @@
+#  This code is contributed by Riyazul555
+
+#  Time Complexity  =  O(V + E)  where V is the number of vertices and E is the number of edges.
+#  Space Complexity =  O(V + E)  where V is the number of vertices and E is the number of edges.
+
+def euler(nod, G, e, viz):
+    while G[nod]:
+        it = G[nod].pop()
```

Comment:
This is C++ style for an iterator, consider using two variables for the next node and the edge.